app/assets/stylesheets/templates/_messages.scss

```diff
@@ -2,8 +2,95 @@
   .slab--header {
     padding: {
       top: 1em;
-      bottom: 1em;
+      bottom: .5em;
+    }
+    h2, p {
+    	margin-bottom: 0;
     }
   }
-
 }
+.message--container {
+	display: flex;
+	flex-wrap: wrap;
+  	justify-content: space-between;
+  	max-width: 750px;
+  	margin-bottom: 1em;
+	.message--outbound {
```

Comment:
style nit: I'd recommend leaving an empty line above new CSS rules, this will make the file easier to read and navigate.